apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -254,3 +272,45 @@ export const fetchWorkflowEventDecision = async ({
 
   return handleZodError(error, workflow);
 };
+
+export const createWorkflow = async ({
```

Comment:
`createWorkflow` is a name used in the system as the way to use our SDKs, this might cause confusion.